test/run_notebooks.sh

```diff
@@ -4,13 +4,23 @@ RED='\033[0;31m'
 GREEN='\033[0;32m'
 NC='\033[0m' # No Color
 
-## Returns the correct image name.  If the test name's
-## environmental variable exists, return that, otherwise return the default
-## value for the image.
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+
+## Returns the correct image name to run a notebook.
+## If the test name's environmental variable exists, return that.
+## otherwise:
+##   if the second argument passed is 'true' return the tag value for the image read from the version.txt file.
+##   else returns 'latest'
 function image_name () {
     base="regression-tests-$1"
+    if [ "$2" = true ]; then
+	recent_tag=$(<"$SCRIPT_DIR/$1/version.txt")
```

Comment:
The two lines for `recent_tag=` are also tabs instead of spaces. (Only spotted because in my editor the line up with the `if`, `else` and `fi` lines.